value_test.go

```diff
@@ -473,6 +474,27 @@ func TestValueFunction(t *testing.T) {
 
 }
 
+func TestNewStringFromByteArray(t *testing.T) {
+	t.Parallel()
+	ctx := v8.NewContext()
+	iso := ctx.Isolate()
+
+	inputString := "CN7ySxWjjWNpbYPB1n/TBR8avujZS2cHdXRR5ZM7Fi6QBjlVzBqPu0CI9pQXcW9fvbMXdqU+57XY/QdGozJT19+gbQDM0ZQVzjwqtpyLorcPHjMqum+7dHD6XF5cXo3NZlKGsxcnvSVyClBDU5M1dUCe8bB9yV0wVM6ge+0WAmTX2GYbncilTjDw0bSJI1Z+71NT8UQCfmimKhVxJiKrnkaTrTw2Ma/1I2w4Dny3cRlFtCtob9cvNOeeIm8HtQoi/7HXoE0uFr1C39OL2hCC1TJsxX94djtNFqd9aUOPYrwT+zErSokSvbNYS5WpEjEpRJze9+TCV9NLmqCnARK4Bw"
+	byts, _ := base64.RawStdEncoding.DecodeString(inputString)
+
+	expected := "ÞòK£cimÁÖÓ¾èÙKgutQå;.9UÌ»@öqo_½³v¥>çµØýF£2S×ß m"
+	val, err := v8.NewStringFromByteArray(iso, byts)
+	if err != nil {
+		t.Errorf("expected nil but got error %#v", err)
+	}
+	if !val.IsString() {
+		t.Errorf("expected string but got %s", reflect.TypeOf(val))
+	}
+	if val.String() != expected {
+		t.Errorf("expected not same as actual")
+	}
```

Comment:
Shouldn't this also work for `v8.NewValue(expected)`, in which case this isn't testing the distinguishing behaviour of this new method, which is that the bytes aren't interpreted as UTF-8 in the JS string.  Since there isn't a v8go.String type, a workaround would be to test this in JS (e.g. getting the length and making sure it counts each byte in a UTF-8 encoded byte sequence)